### ISSUE TITLE
Discoverability system

### ISSUE THREAD

**Issue by cathoderaydude** [WinWorldPC/adventure#77]
Release page should link to related items. Site should have "random product" links at top.

**Comment by NattyNarwhal** [comment]
Random could be accessible from outside of a product page. Perhaps it could also bias towards unloved downloads, though I bet the majority of downloads will outnumber them.

**Comment by cathoderaydude** [comment]
Yeah I was thinking Random ought to be a top navbar feature. We could also literally expose two links, one for "random popular app" and one for "random unknown object"

**Comment by NattyNarwhal** [comment]
What about doing it as a random sort order for search? That way, users could filter on criteria (like random game, etc) for their randomness.
